src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -57,7 +57,7 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
     }
 
     TermControl::TermControl() :
-        TermControl(Settings::TerminalSettings{}, TerminalConnection::ITerminalConnection{ nullptr })
+        TermControl(Settings::IControlSettings{ nullptr }, TerminalConnection::ITerminalConnection{ nullptr })

```

Comment:
This is a change I'm unsure of. Now that `TerminalSettings` is in TermApp, it doesn't make sense to take a dependency on TermApp. So I had to change this to `IControlSettings`.

Reply:
You know what, we're not event using that ctor anywhere. I'd just pull it from the idl/h/cpp